firebase-firestore/src/main/java/com/google/firebase/firestore/remote/GrpcCallProvider.java

```diff
@@ -75,23 +71,19 @@ public static void overrideChannelBuilder(
     this.asyncQueue = asyncQueue;
 
     TaskCompletionSource<ManagedChannel> channelReady = new TaskCompletionSource<>();
-    this.channelTask = channelReady.getTask();
 
-    // We execute network initialization on a separate thread to not block operations that
+    // We execute network initialization on a separate queue to not block operations that
```

Comment:
Not certain why you changed thread => queue.  I think "thread" was and is more accurate.

Reply:
Undone. Not sure what the best term here is since the thread might already exist.